### PR TITLE
Add support for Get Workitem query tags

### PR DESCRIPTION
**Pull request by bcarthic**
## Description
Add support to get workitem query tags

## Related issues
Addresses [issue #].

## Testing
Added unit tests and integration tests

### CODE REVIEW

File: src/Microsoft.Health.Dicom.SqlServer/Features/Workitem/SqlWorkitemStoreV8.cs [hunk 1]
```diff
@@ -16,6 +16,8 @@
 using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
 using Microsoft.Health.SqlServer.Features.Client;
 using Microsoft.Health.SqlServer.Features.Storage;
+using System.Data;
```
Comment (jovinson-ms) [inline src/Microsoft.Health.Dicom.SqlServer/Features/Workitem/SqlWorkitemStoreV8.cs:19]: need to sort

File: test/Microsoft.Health.Dicom.Tests.Integration/Persistence/WorkitemTests.cs [hunk 2]
```diff
@@ -38,9 +38,43 @@ public async Task WhenValidWorkitemIsCreated_CreationSucceeds()
...
+            var queryTags = new List<QueryTag>()
+            {
+                new QueryTag(new ExtendedQueryTagStoreEntry(2, tag2.GetPath(), tag2.GetDefaultVR().Code, null, QueryTagLevel.Instance, ExtendedQueryTagStatus.Ready, QueryStatus.Enabled,0)),
+            };
+
+            long workitemKey = await _fixture.IndexWorkitemStore.AddWorkitemAsync(DefaultPartition.Key, dataset, queryTags, CancellationToken.None);
+
+            await _fixture.IndexWorkitemStore.DeleteWorkitemAsync(DefaultPartition.Key, workitemUid, CancellationToken.None);
+
+            // Try adding it back again, if this succeeds, then assume that Delete operation has succeeded.
```
Comment (jovinson-ms) [inline test/Microsoft.Health.Dicom.Tests.Integration/Persistence/WorkitemTests.cs:65]: can't we get to check it's not there?

**Review by jovinson-ms — APPROVED** [review]

**Review by alishahahmed — APPROVED** [review]

File: test/Microsoft.Health.Dicom.Tests.Integration/Persistence/WorkitemTests.cs (hunk 2 above)
Comment (bcarthic) [reply test/Microsoft.Health.Dicom.Tests.Integration/Persistence/WorkitemTests.cs:65]: Currently we didnt implement GetWorkitem, so this is a temp code, once we have getworkitem done, we can remove it. I will have add a comment

File: src/Microsoft.Health.Dicom.SqlServer/Features/Workitem/SqlWorkitemStoreV8.cs (hunk 1 above)
Comment (bcarthic) [reply src/Microsoft.Health.Dicom.SqlServer/Features/Workitem/SqlWorkitemStoreV8.cs]: Done

**Pull request merged**